README.md

```diff
@@ -678,17 +679,73 @@ test(
   withBrowser(async ({ utils, screen }) => {
     // element is an ElementHandle (pointer to an element in the browser)
     const element = await screen.getByText(/button/i);
-    // we can pass element into runJS and the default exported function can access it as an Element
+    // we can pass element into runJS and access it as an Element via import.meta.pleasantestArgs
     await utils.runJS(
       `
-        export default (element) => console.log(element);
+      const [element] = import.meta.pleasantestArgs;
+      console.log(element);
       `,
       [element],
     );
   }),
 );
 ```
 
+The code string passed to `runJS` is also a module, and it can export values to make them available in Node. `runJS` returns a Promise resolving to the exports from the module that executed in the browser. Each export is wrapped in a `[`JSHandle`](https://pptr.dev/#?product=Puppeteer&version=v13.5.2&show=api-class-jshandle) (a pointer to an in-browser JS object), so that it can be passed back into the browser if necessary, or deserialized in Node using `.jsonValue()`.
```

Comment:
```suggestion
The code string passed to `runJS` is also a module, and it can export values to make them available in Node. `runJS` returns a Promise resolving to the exports from the module that executed in the browser. Each export is wrapped in a [`JSHandle`](https://pptr.dev/#?product=Puppeteer&version=v13.5.2&show=api-class-jshandle) (a pointer to an in-browser JS object), so that it can be passed back into the browser if necessary, or deserialized in Node using `.jsonValue()`.
```